src/dialog/dialog.component.ts

```diff
@@ -63,13 +64,15 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 	/**
 	 * `Subscription` used to update placement in the event of a window resize.
 	 */
-	protected resizeSubscription: Subscription;
+	protected resizeSubscription: Subscription = new Subscription();
```

Comment:
you can remove `: Subscription` now that the type is implicitly defined through `new Subscription()`